README.md

```diff
@@ -39,21 +39,30 @@ A curated list of computer history videos, documentaries and related folklore ma
 - [The KGB, the Computer, and Me](https://www.youtube.com/watch?v=EcKxaq1FTac) (1990) - With computer scientist Clifford Stoll <sup>[8.3/10](http://www.imdb.com/title/tt0308449/)</sup>
 - [Triumph of the Nerds: The Rise of Accidental Empires](https://en.wikipedia.org/wiki/Triumph_of_the_Nerds) (1996) - History of the personal computer <sup>[8.5/10](http://www.imdb.com/title/tt0115398/)</sup>
 - [Code Rush](https://www.youtube.com/watch?v=4Q7FTjhvZ7Y) (2000) - The story of Netscape and the birth of Mozilla <sup>[7.3/10](http://www.imdb.com/title/tt0499004/)</sup>
+- [The Code](https://www.youtube.com/watch?v=XMm0HsmOTFI) (2001) - The Histroy of Linux & FOSS <sup>[7.6/10](http://www.imdb.com/title/tt0315417/)</sup>
 - [Revolution OS](https://www.youtube.com/watch?v=jw8K460vx1c) (2001) - The story of GNU/linux and Open Source Movement <sup>[7.3/10](http://www.imdb.com/title/tt0308808/)</sup>
 - [The Secret History Of Hacking](https://www.youtube.com/watch?v=PUf1d-GuK0Q) (2001) - History of early hackers <sup>[7.5/10](http://www.imdb.com/title/tt2335921/)</sup>
 - [BBS The Documentary](https://www.youtube.com/playlist?list=PLgE-9Sxs2IBVgJkY-1ZMj0tIFxsJ-vOkv) (2005) - History of the Bulletin Board System <sup>[8.6/10](http://www.imdb.com/title/tt0460402/)</sup>
+- [Hackers Are People Too ](https://www.youtube.com/watch?v=7jciIsuEZWM) (2008) <sup>[6.1/10](http://www.imdb.com/title/tt1279942/)</sup>
 - [Something Ventured](http://www.imdb.com/title/tt1737747/) (2011) - The history of early venture capital <sup>[7.0/10](http://www.imdb.com/title/tt1737747/)</sup>
+- [The Codebreakers](https://www.youtube.com/watch?v=Zc-hlV2xbSg) (2011) - Documentary about how poor countries are using free and open source software (FOSS) for development purposes. <sup>[7.1/10](http://www.imdb.com/title/tt2119396/)</sup>
```

Comment:
Link movie linked to in the IMDb link isn't the right one. I couldn't find it on IMDb. The video it self links to this wikipedia article: https://en.wikipedia.org/wiki/The_Code-Breakers

According to the wiki article it's made in 2006. Please place it accordingly and just remove the IMDb link.